### ISSUE TITLE
Image not imported

### ISSUE THREAD

**Issue by lsantaniello** [etruel/wpematico#59]
Hi, I need to use "https://www.corrieredellosport.it/rss/" or rimilar RSS Feed.

This feed use this interface for each item:
```
<item>
	<title>My Title</title>
	<description>My description</description>
	<link>My Link</link>
	<category>My Category</category>
	<pubDate>Thu, 31 Aug 2023 09:44:19 GMT</pubDate>
	<media:content url="https://myimg.jpg" type="image/jpg" expression="full" width="600" height="400">
		<media:copyright/>
	</media:content>
</item>
```

The image is content into "media:content" tag but It is not imported.

I configured "Options for Images" but I think that the plugin do not support this interface. Is it correct?

How can solve the problem?

Thanks
L

**Comment by manuelge** [comment]
Hi ,

We are sorry to hear that you're having trouble getting the featured images, please allow me to explain.

**WPeMatico FREE** fetches the content directly from the **<content>** or **<description>** tags of the feed items, and, the image must be there for it to fetch them.

On the other hand the [Professional](https://etruel.com/downloads/wpematico-professional/) addon allows to get the images included in **<enclosure>** or **<media>** tags of the feed items, you may want to take a look at all its features.

We invite you to take a look at this article for more information: [https://www.wpematico.com/set-images-featured-wpematico/](https://www.wpematico.com/set-images-featured-wpematico/)

**Comment by Mark-AAAA** [comment]
Do you have FIFU installed ?

**Comment by alexanderchmielewski** [comment]
Any chance to get an enhancement on this? I'm trying to get a RSS Slideshow to work which exactly uses yahoo's media namespace for each slide.

I altered above's example to demonstrate the use-case

`<item>
	<title>My Title</title>
	<description>My description</description>
	<link>My Link</link>
	<category>My Category</category>
	<pubDate>Thu, 31 Aug 2023 09:44:19 GMT</pubDate>
	
        <media:content url="https://myimg.jpg" type="image/jpg" expression="full" width="600" height="400">
		<media:copyright/>
	</media:content>
       	
        <media:content url="https://myimg.jpg" type="image/jpg" expression="full" width="600" height="400">
		<media:copyright/>
	</media:content>
        
        <media:content url="https://myimg.jpg" type="image/jpg" expression="full" width="600" height="400">
		<media:copyright/>
	</media:content>
</item>`

**Comment by manuelge** [comment]
Hi @alexanderchmielewski, thank you for your question.

As I mentioned in my previous comment, **WPeMatico FREE** fetches the images and content from the **`<content>`** or **`<description>`** tags of the feed items.

To get images from enclosure/media <tags> you will need the **Professional** addon features.

I invite you to take a look at this article where you will find more information about featured images and how to get them.

[https://www.wpematico.com/set-images-featured-wpematico/](https://www.wpematico.com/set-images-featured-wpematico/)

Regards,
Manuel

**Comment by alexanderchmielewski** [comment]
Thank you for your quick response @manuelge. 
I do have professional but it is not working with the above example.

**Comment by etruel** [comment]
We are seeing this simplepie issue on a private repository because it is a pro feature.  Anyway we planned fix it for 2.7 version 

https://github.com/simplepie/simplepie/issues/585#issuecomment-392021849

https://github.com/WordPress/WordPress/blob/ffb401296259cc20eed753bf0f934221df45e32e/wp-includes/SimplePie/Item.php#L1085 

https://github.com/etruel/wpematico_professional/issues/25

**Comment by alexanderchmielewski** [comment]
Thank you so much for that insight. Is there any ETA on 2.7?

Cheers
